docs/src/documentation/03-components/07-interaction/stepper/03-accessibility.mdx

```diff
@@ -16,7 +16,7 @@ The following props provide additional information to screen readers:
 
 - The `titleIncrement` prop allows you to specify an `aria-label` attribute for the increment icon button in the Stepper (StepperStateless) component.
 
-- The `ariaLabelledBy` prop allows you to specify an `aria-labelledby` attribute for the Stepper component. This attribute references the ID of the element that labels the Stepper (StepperStateless), ensuring that screen readers announce the label correctly.
```

Comment:
If you will keep the `ariaLabelledBy`, I would add here that it's labelling just the value, not buttons or the whole component, just to be more clear.